build.gradle

```
@@ -82,7 +82,7 @@ configurations.all {
         force 'commons-cli:commons-cli:1.3.1'
         force 'org.apache.httpcomponents:httpcore:4.4.12'
         force "org.apache.commons:commons-lang3:3.4"
-        force "org.springframework:spring-core:5.3.26"
+        force "org.springframework:spring-core:5.3.28"
         force "org.springframework:spring-expression:5.3.27"
```

ah ... sorry missed the this line

```suggestion
        force "org.springframework:spring-expression:5.3.28"
```

apologies :(

---

No worries! Thanks for catching these!